arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -18,26 +18,18 @@ namespace ArangoDBNetStandard.Transport.Http
     /// </remarks>
     public class HttpApiTransport : IApiClientTransport
     {
-        private readonly HttpClient _client;
-        private HttpContentType _contentType;
-
-        private static readonly Dictionary<HttpContentType, string> _contentTypeMap =
-            new Dictionary<HttpContentType, string>
-            {
-                [HttpContentType.Json] = "application/json",
-                [HttpContentType.VPack] = "application/x-velocypack"
-            };
+        private readonly HttpClientConfig _clientConfig;
 
         /// <summary>
         /// Create <see cref="HttpApiTransport"/> from an existing <see cref="HttpClient"/> instance.
         /// </summary>
-        /// <param name="client">Existing HTTP client instance.</param>
+        /// <param name="clientConfig">Existing HTTP client configuration instance.</param>
         /// <param name="contentType">Content type to use in requests.
         /// Used to set Content-Type and Accept HTTP headers.</param>
-        public HttpApiTransport(HttpClient client, HttpContentType contentType)
+        public HttpApiTransport(HttpClientConfig clientConfig, HttpContentType contentType)
```

Comment:
Hmm, I would rather keep the interface changes at the minimum, so that the applications that already use this client don't need to be adapted. What are your thoughts?